src/objects.js

```diff
@@ -1,43 +1,58 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return !!object[property];
 };
 
 const isOver65 = person => {
-  // your code here
+  return person.age>65;
 };
 
 const getAges = people => {
-  // your code here
-};
+  const array = [];
+  // eslint-disable-next-line no-plusplus
```

Comment:
You should try avoid disabling eslint rules whenever possible. Instead, try to fix what's causing the issue. In this case, it wants you to use `i += 1` instead of `i++`.